src/mocks/data/fileSubmissions.ts

```diff
@@ -28,6 +28,20 @@ const getIssues = (status: string, route: string): string[] => {
   return [];
 };
 
+const createSentBy = () => {
+  const prefix = ["PH", "ST", "LB", "CO", "HO", "PR"];
+
+  const arrayLength = prefix.length;
+
+  const randomIndex = Math.floor(Math.random() * arrayLength);
+
+  return `${prefix[randomIndex]}-LA`;
+};
```

Comment:
Update to this: 

const createSentBy = () => {
  const prefix = ["PH", "ST", "LB", "CO", "HO", "PR"];
  const randomPrefix = faker.helpers.arrayElement(prefix)

  return `${randomPrefix}-LA`;
}